containers/ecr-viewer/src/app/admin/user/UserForm.tsx

```diff
@@ -0,0 +1,278 @@
+
+  return (
+    <FormPageContent
+      action={`${action} user`}
+      formValid={valid}
+      submitAction={async () => {
+        await submitAction(email, userType, selectedPrograms);
+      }}
+      successRoute="/admin/user"
+    >
+      <EmailFieldSet email={email} setEmail={setEmail} />
+      <UserTypeFieldSet userType={userType} setUserType={setUserType} />
+      <ProgramFieldSet
+        programs={programs}
+        setPrograms={setPrograms}
+        numProgramsSelected={numProgramsSelected}
+      />
+    </FormPageContent>
+  );
+};
+
+const EmailFieldSet = ({
+  email,
+  setEmail,
+}: {
+  email: string;
+  setEmail: (n: string) => void;
+}) => {
+  return (
+    <FieldSet legend="AD Email">
+      <span>
+        Required fields are marked with an asterisk (<RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        AD EMAIL
+        <RequiredMarker />
+        <TextInput
+          type="text"
+          required={true}
+          id="email"
+          name="email"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const UserTypeFieldSet = ({
+  userType,
+  setUserType,
+}: {
+  userType: UserType;
+  setUserType: (n: UserType) => void;
+}) => {
+  return (
+    <FieldSet legend="Select user type">
+      <span>
+        User type will apply to all users added. These can be edited later (
+        <RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        USER TYPE
+        <RequiredMarker />
+        <Radio
+          label="Admin"
+          type="radio"
+          required={true}
+          name="userType"
+          id="userType-admin"
+          value="admin"
+          checked={userType === "admin"}
+          onChange={(e) => setUserType(e.target.value as UserType)}
+        />
+        <Radio
+          label="Standard"
+          type="radio"
+          required={true}
+          name="userType"
+          id="userType-standard"
+          value="standard"
+          checked={userType === "standard"}
+          onChange={(e) => setUserType(e.target.value as UserType)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const ProgramFieldSet = ({
+  programs,
+  setPrograms,
+  numProgramsSelected,
+}: {
+  programs: FormProgram[];
+  setPrograms: (c: FormProgram[]) => void;
+  numProgramsSelected: number;
+}) => {
+  const [expandedPrograms, setExpandedPrograms] = useState<
+    Record<string, boolean>
+  >(valsToBoolean(programs, true));
+
+  const accordionItems: AccordionItem[] = programs.map((program) => {
+    const name = program.name;
+    const conditions = program.conditions;
+    const numConditions = conditions.length;
+
+    return {
+      title: (
+        <div className="display-flex flex-justify flex-align-center">
+          <div>
+            <React.Fragment key={`program-${program.uuid}`}>
+              <Checkbox
```

Comment:
you're going to run into accessibility trouble here. @echinelo-skylight this is another instance of accordion+checkbox in accordion header